libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -38,23 +38,31 @@ extern const AP_HAL::HAL& hal;
 const AP_Param::GroupInfo GCS::var_info[] {
     // @Param: _SYSID
     // @DisplayName: MAVLink system ID of this vehicle
-    // @Description: Allows setting an individual MAVLink system id for this vehicle to distinguish it from others on the same network
+    // @Description: Allows setting an individual MAVLink system id for this vehicle to distinguish it from others on the same network.
     // @Range: 1 255
     // @User: Advanced
     AP_GROUPINFO("_SYSID",    1,     GCS,  sysid,  MAV_SYSID_DEFAULT),
 
     // @Param: _GCS_SYSID
     // @DisplayName: My ground station number
-    // @Description: This controls whether packets from other than the expected GCS system ID will be accepted
+    // @Description: This sets what MAVLink source system IDs are accepted for GCS failsafe handling, RC overrides and manual control. When MAV_GCS_SYSID_HI is less than MAV_GCS_SYSID then only this value is considered to be a GCS. When MAV_GCS_SYSID_HI is greater than MAV_GCS_SYSID then the range of values between MAV_GCS_SYSID and MAV_GCS_SYSID_HI (inclusive) are all treated as valid GCS MAVLink system IDs
```

Comment:
So there's a case missing here...

Reply:
well, really there isn't, but I've added in the "or equal to" even though I think it is clear without it and makes it more wordy